walsamlee/politico
# #163737817 Consume Create-Office Endpoint

#### What does this PR do?
Add `create-party` API endpoint consumption for `UI`
#### Description of Task to be completed?
Have `create party` endpoint powering `create party` page on the `UI`
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ch-create-office-163737817`
- Visit`https://walsamlee.github.io/politico/admin/createoffice.html` on the browser 
#### What are the relevant pivotal tracker stories?
#163737817
